src/main/java/mods/railcraft/common/blocks/tracks/TrackSwitchBase.java

```diff
@@ -128,6 +143,56 @@ public void onNeighborBlockChange(Block block) {
         }
         super.onNeighborBlockChange(block);
     }
+    
+    /**
+     * A utility method for writing out UUID's to NBT 
+     */
+    private void writeUUIDToNBT(String key, UUID uuid, NBTTagCompound data) {
```

Comment:
Ah, sweet. Yup, I had been looking for classes like NBTTools or NBTUtils. Never thought to look there.